scripts/test/large.py

```diff
@@ -0,0 +1,88 @@
+from modules import utils
+from modules.logger import log
+from unittest import TextTestRunner
+
+import os
+import sys
+import unittest
+
+
+class ExecCollectorLargeTest(unittest.TestCase):
+
+    def setUp(self):
+        # set and download required binaries (snapd, snapctl, plugins)
+        self.binaries = utils.set_binaries()
+        utils.download_binaries(self.binaries)
+
+        log.debug("Starting snapd")
+        self.binaries.snapd.start()
+        if not self.binaries.snapd.isAlive():
+            self.fail("snapd thread died")
+
+        log.debug("Waiting for snapd to finish starting")
+        if not self.binaries.snapd.wait():
+            log.error("snapd errors: {}".format(self.binaries.snapd.errors))
+            self.binaries.snapd.kill()
+            self.fail("snapd not ready, timeout!")
+
+    def test_exec_collector_plugin(self):
+        # load exec collector
+        loaded = self.binaries.snapctl.load_plugin("snap-plugin-collector-exec")
+        self.assertTrue(loaded, "exec collector loaded")
+
+        # load mock file publisher
+        cmd = '{} plugin load {}'.format(os.path.join(self.binaries.snapctl._dir, self.binaries.snapctl.name), "/home/snap-plugin-publisher-mock-file")
+        log.debug("snapctl load plugin {}".format(cmd))
+        out = self.binaries.snapctl._start_process(cmd)
+        log.debug("plugin loaded? {}".format("Plugin loaded" in out))
+
+        # check available metrics, plugins and tasks
+        metrics = self.binaries.snapctl.list_metrics()
+        plugins = self.binaries.snapctl.list_plugins()
+        tasks = self.binaries.snapctl.list_tasks()
+        self.assertGreater(len(metrics), 0, "Metrics available {} expected {}".format(len(metrics), 0))
+        self.assertEqual(len(plugins), 2, "Plugins available {} expected {}".format(len(plugins), 1))
```

Comment:
```python
"Plugins available {} expected {}".format(len(plugins), 1)
```
will result in ```Plugins available 2 expected 1" which is wrong, since you are loading publisher and collector. Expected value in assert is correct though.